bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCacqCam.html

```diff
@@ -316,65 +312,70 @@
+          <td><input name="ImageQuality" value="PERCENT_70" type="radio" /> 70%/Good</td>
+          <td><input name="ImageQuality" value="PERCENT_85" type="radio" checked/> 85%/Poor</td>
+          <td><input name="ImageQuality" value="ANY"        type="radio" /> Any</td>
       </tr>
       <tr>
-          <td><b>Sky transparency (cloud cover):</b></td>
-          <td><input name="CloudCover" value="PERCENT_50" type="radio" checked /> 50%/Clear</td>
-          <td><input name="CloudCover" value="PERCENT_70" type="radio" /> 70%/Cirrus</td>
+          <td><b>Cloud Cover:</b></td>
+          <td><input name="CloudCover" value="PERCENT_50" type="radio" /> 50%/Clear</td>
+          <td><input name="CloudCover" value="PERCENT_70" type="radio" checked /> 70%/Cirrus</td>
           <td><input name="CloudCover" value="PERCENT_80" type="radio" /> 80%/Cloudy</td>
-          <td><input name="CloudCover" value="ANY" type="radio" /> Any</td>
+          <td><input name="CloudCover" value="ANY"        type="radio" /> Any</td>
       </tr>
       <tr>
-          <td><b>Sky transparency (water vapour):</b></td>
+          <td><b>Water Vapor:</b></td>
           <td><input name="WaterVapor" value="PERCENT_20" type="radio" /> 20%/Low</td>
           <td><input name="WaterVapor" value="PERCENT_50" type="radio" /> 50%/Median</td>
-          <td><input name="WaterVapor" value="PERCENT_80" type="radio" checked /> 80%/High</td>
-          <td><input name="WaterVapor" value="ANY" type="radio" /> Any</td>
+          <td><input name="WaterVapor" value="PERCENT_80" type="radio" /> 80%/High</td>
+          <td><input name="WaterVapor" value="ANY"        type="radio" checked /> Any</td>
       </tr>
       <tr>
-          <td><b>Sky background:</b></td>
+          <td><b>Sky Background:</b></td>
           <td><input name="SkyBackground" value="PERCENT_20" type="radio" /> 20%/Darkest</td>
           <td><input name="SkyBackground" value="PERCENT_50" type="radio" /> 50%/Dark</td>
           <td><input name="SkyBackground" value="PERCENT_80" type="radio" checked /> 80%/Grey</td>
-          <td><input name="SkyBackground" value="ANY" type="radio" /> Any/Bright</td>
+          <td><input name="SkyBackground" value="ANY"        type="radio" /> Any/Bright</td>
       </tr>
       <tr>
-          <td><b>Typical air mass during observation:</b></td>
+          <td><b>Airmass:</b></td>
           <td><input name="Airmass" value="1.2" type="radio" /> &lt;1.2</td>
           <td><input name="Airmass" value="1.5" type="radio" checked /> 1.5</td>
           <td><input name="Airmass" value="2.0" type="radio" /> 2.0</td>
       </tr>
 
     <tr>
       <td colspan="6">
-          <p align="right"><input value="Calculate" type="submit" style="background-color: gold; padding: 3px 8px;"></p>
+          <p align="right">
+              <input value="Calculate" type="submit" style="background-color: gold; padding: 3px 8px;">
+          </p>
       </td>
     </tr>
-
+      </tbody>
```

Comment:
This seems overindented.

Reply:
Agreed!  I used the very handy "Auto-Indent" feature in IntelliJ and the result looks much nicer!  I'll update the PR shortly.